### PR TITLE
update gitignore

### PR DESCRIPTION
**Pull request by Kozbeli**
Refactoring, linting and formating.

### CODE REVIEW

File: backend/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (eherra) [inline backend/.prettierrc:1]: good addition! 💯

File: backend/src/routes/weaviate.ts [hunk 2]
```diff
@@ -1,126 +1,95 @@
...
-      .withNearText({
-        concepts: [entities],
-      })
-      .withFields("chunk_index")
-      .withLimit(4)
-      .do();
+    )
 
-    console.log(JSON.stringify(result, null, 2));
+    const entities = response.data.choices[0].message.content.trim()
```
Comment (eherra) [inline backend/src/routes/weaviate.ts:78]: Nitpick:
To get a bit more robustness, I would consider checking nulls here:
`response?.data?.choices[0]?.message?.content.trim()` etc

OpenAI's API not the most reliable service

**Review by eherra — APPROVED** [review]
goodd stuff

**Review by Le36 — APPROVED** [review]
excellent work

File: backend/src/routes/weaviate.ts (hunk 2 above)
Comment (Kozbeli) [reply backend/src/routes/weaviate.ts:78]: Good attention, now resloved...

**Pull request merged**